ecs-agent/csiclient/dummy_csiclient.go

```diff
@@ -1,55 +0,0 @@
-// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
@xxx0624 Just a heads up regarding this change.

Reply:
Added it back 😅